### PR TITLE
Update posix_sitl_default to px4_sitl_default for px4 build

### PR DESCRIPTION
**Pull request by dakejahl**
Changed the naming from `posix` to `px4` to match the new naming in `PX4/Firmware` `master`

### CODE REVIEW

**Comment by dakejahl** [comment]
Lol at home trying to do this through github. I'll change all the posix's to px4's tomorrow as I continue working through this. :smile:

**Comment by dakejahl** [comment]
Removed `model` as an arg to `global_planner_sitl_mavros.launch`.

File: global_planner/launch/global_planner_stereo.launch [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <arg name="use_three_point_msg" value="$(arg use_three_point_msg)" />
         <arg name="use_stereo_simulation" value="true" />
         <arg name="point_cloud_topic" value="/stereo/points2" />
-        <arg name="model" value="iris_stereo_camera" />
+        <!-- <arg name="model" value="iris_stereo_camera" /> -->
```
Comment (mrivi) [inline global_planner/launch/global_planner_stereo.launch]: could you remove this instead of commenting it out?

Comment (nicovanduijn) [inline global_planner/launch/global_planner_stereo.launch]: Even better: change the `global_planner_sitl_mavros.launch` and `mavros_sitl.launch` to use the argument and pass it down. (Currently `mavros_sitl.launch` hardcodes the vehicle to 'iris_depth_camera')

**Comment by nicovanduijn** [comment]
Can we merge this? @jkflying @mrivi

**Pull request merged**